src/jobs/test-nut.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+description: >
+  Runs NUTs for package.  Set environment: TESTKIT_EXECUTABLE_PATH:whatever to use an executable besides bin/run
+
+parameters:
+  node_version:
+    description: version of node to run tests against
+    type: string
+    default: "12"
+  os:
+    description: operating system to run tests on
+    type: enum
+    enum: ['linux', 'windows']
+    default: 'linux'
+  sfdx_version:
+    description: 'By default, the latest version of the standalone CLI will be installed. To install via npm, supply a version tag such as "latest" or "6".'
+    default: ''
+    type: string
+
+executor:
+  name: << parameters.os >>
+
+steps:
+  - install-node:
+      version: <<parameters.node_version>>
+      os: <<parameters.os>>
+  - install-sfdx:
+      version: <<parameters.sfdx_version>>
+      os: <<parameters.os>>
```

Comment:
Again, I don't think `install-sfdx` should behave differently based on the os

Reply:
another option would be to have `install-sfdx-npm` and `install-sfdx-artifact` and make that a nut parameter.  

Definitely worth a look at the user story @shetzel wrote.